Sources/Noora/Utilities/Terminal.swift

```diff
@@ -9,15 +9,23 @@ import Foundation
 public protocol Terminaling {
     var isInteractive: Bool { get }
     var isColored: Bool { get }
+    var size: TerminalSize? { get }
```

Comment:
As raised by @pepicrft [here](https://github.com/tuist/Noora/pull/195#discussion_r1969284368), let's make this into a method.